src/PackageUtilities.js

```diff
@@ -83,4 +83,31 @@ export default class PackageUtilities {
     }
     return packages;
   }
+
+  static filterNonDependentPackages(packages, scope) {
+    const filteredGraph = PackageUtilities.getPackageGraph(packages);
+
+    let dependentPackages = [];
+    let fringe = [];
+    fringe.push(scope);
+
+    let dependencies;
+    while (fringe.length !== 0) {
+      let pkg = fringe.shift();
+
+      dependencies = filteredGraph.get(pkg).dependencies;
+
+      dependencies.forEach((dependency) => {
+        if (!~dependentPackages.indexOf(dependency) && !~fringe.indexOf(dependency)) {
```

Comment:
Yep. It's definitely less cryptic than `!~`. 

Any plans to use ES7? It includes Array.prototype.includes which would be the ideal solution to checking for existence.